drivers/serial/Kconfig.nrfx_uart_instance

```diff
@@ -3,9 +3,39 @@
 # Copyright (c) 2023 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+if $(nrfx_uart_num) = 0
+config UART_$(nrfx_uart_num)_NRF_UART
+	def_bool HAS_HW_NRF_UART$(nrfx_uart_num)
+	depends on DT_HAS_NORDIC_NRF_UART_ENABLED
+	select NRF_UART_PERIPHERAL
+	help
+	  Enable nRF UART without EasyDMA on port $(nrfx_uart_num).
+endif
+
+config UART_$(nrfx_uart_num)_NRF_UARTE
+	def_bool HAS_HW_NRF_UARTE$(nrfx_uart_num)
+	depends on DT_HAS_NORDIC_NRF_UARTE_ENABLED || DT_HAS_NORDIC_NRF_UARTE_V2_ENABLED
+	select NRF_UARTE_PERIPHERAL
+	select NRFX_UARTE$(nrfx_uart_num) if !UART_NRFX_LEGACY_SHIM
+	select NRFX_UARTE if !UART_NRFX_LEGACY_SHIM
+	help
+	  Enable nRF UART with EasyDMA on port $(nrfx_uart_num).
+
+if UART_$(nrfx_uart_num)_NRF_UART || UART_$(nrfx_uart_num)_NRF_UARTE
```

Comment:
These options were removed in aa7d675935517cce0668fa12491602c321b1a0bf and should not be reintroduced in this PR.

Reply:
Removed